docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+#### Running the OSCAL site locally
+
+Running Hugo with its own server locally is a convenient way of previewing site contributions as they will actually appear. Not all content (especially if unproblematic Markdown) needs to be previewed, but it is a good way to intercept display problems early.
+
+This is documented in the repository [docs directory readme](https://github.com/usnistgov/OSCAL/tree/main/docs).
+
+From a Linux shell, use command
+
+```sh
+$ hugo server -v --debug --minify
+```
+
+to start the server on port 1313, viewable at `http://localhost:1313/OSCAL/`.
+
+Note that (as detailed in the readme) to browse the entire site with all contents, including generated contents, additional scripting may need to be run prior to starting the server.
+
+## Style
+
+### Accessibility
+
+Since materials for the web site are authored using generic encoding (Markdown) and produced with an automated production pipeline, we are able to encapsulate and manage page design within our build. This takes accessibility and accessibility compliance (such as ADA) out of the hands of authors, for the most part.
+
+Please be so kind as to provide `alt` values for all images to save us effort in meeting Accessibility benchmarks. Otherwise, authors should be able to rely on the publishing process to handle technical requirements related to accessibility.
+
+At the same time, the builders of the back end support for these publications will be interested to know of any accessibility-related requirements that are not already adequately addressed.
+
+If publishing a PDF and not simply web resources through the Pages platform, please coordinate and keep in mind requirements in this area for accessible PDFs (PDFa format).
+
+- [ADA.gov resources](https://beta.ada.gov/web-guidance/)
+- [US Web Design System Accessibility page](https://designsystem.digital.gov/documentation/accessibility/)
+- Also search the NIST Intranet for applicable guidance
+
+### Acronyms
+
+The general rule is to spell out acronyms on first use. Keep in mind not all readers are experts in every technology or domain being discussed.
+
+There are cases where the general rule does not apply, because an acronym has become a well-known term of art. <q>HTML</q> does not have to be explained on a page that assumes web technology as context. If an acronym does not actually provide information to anyone in the audience, consider whether it is necessary to explain it. As always, context should be the determining factor. Another example is <q>PR</q> used in reference to git or Github. Similarly this style guide, because it is focused on a web publishing platform, does not explain HTML or CSS, and knowing that HTML expands to <q>Hypertext Markup Language</q> is not important enough to stress, given how easy it is to discover in any case.
```

Comment:
I thought part of what we intended for this PR, re the goals (but not so much the AC of this issue, and I feel I personally dropped the ball on that) is the call out jargon more prominently. 

I know acronyms are not that, but there is a lot of overlap in the case of OSCAL syntax, semantics, NIST terminology and industry terminology (SSP, SAP, SAR, and POA&M definitely come to mind, not just as OSCAL models, but terminology used in US federal cybersecurity and the industry overall).

Could we commit to something like this?

> Please do not use acronyms unless they are commonly used by the OSCAL community in writing and verbal communication. If unsure, we prefer that we only use acronyms that are defined in [the OSCAL Key Terms and Concepts index](https://pages.nist.gov/OSCAL/concepts/terminology/). If not there, [The NIST Cybersecurity Resource Center Glossary](csrc.nist.gov/glossary/), then $X a more general resource, and $Y one more general than that."

Reply:
Nope I think this is too stringent. See above about policing and need/expectation for policing. I do not expect anyone would be able to prioritize checking us all up against this or any other guideline especially one liable to change (or possibly itself in need of maintenance). It is better to describe what good practice is, then demonstrate it.

I can capture those links and write some clarification, however, which I will do.